packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -96,6 +96,17 @@ export function toRemovalInfo(
 	);
 }
 
+/**
+ * Tracks information about each concurrent move operation which affected a segment.
+ * @legacy
+ * @alpha
+ */
+export interface IConcurrentMoveInfo {
```

Comment:
Do we expect external users to implement this interface? Or just observe objects of this type? If the latter, we should probably mark this as `@sealed`. That way we reserve the right to make certain classes of changes to it without them being "breaking".